jdk/src/share/lib/security/java.security-aix

```diff
@@ -703,7 +703,7 @@ jdk.jar.disabledAlgorithms=MD2, MD5, RSA keySize < 1024, \
 # Example:
 #   jdk.tls.disabledAlgorithms=MD5, SSLv3, DSA, RSA keySize < 2048
 jdk.tls.disabledAlgorithms=SSLv3, TLSv1, TLSv1.1, RC4, DES, MD5withRSA, \
-    DH keySize < 1024, EC keySize < 224, 3DES_EDE_CBC, anon, NULL, \
+    DH keySize < 1024, EC keySize < 224, 3DES_EDE_CBC, anon, NULL, ECDH, \
```

Comment:
Please move the `ECDH` addition to a new line to match the [JDK 11 backport](https://github.com/openjdk/jdk11u-dev/commit/5dddf69319480251cdc904cf7a1d4fcd81573bb8) more closely.
```suggestion
    DH keySize < 1024, EC keySize < 224, 3DES_EDE_CBC, anon, NULL, \
    ECDH, \
```

Reply:
This makes future reviews easier.

Reply:
fixed